server/sonar-server/src/main/java/org/sonar/server/issue/index/IssueIndex.java

```diff
@@ -730,6 +722,7 @@ private BoolFilterBuilder createBoolFilter(IssueQuery query) {
           IssueIndexDefinition.FIELD_ISSUE_FUNC_CREATED_AT},
         null)
       .setQuery(QueryBuilders.filteredQuery(QueryBuilders.matchAllQuery(), filter));
+    // .setQuery(boolQuery().must(matchAllQuery()).filter(filter));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)